launch_ros/launch_ros/actions/node.py

```diff
@@ -321,11 +324,17 @@ def node_name(self):
             raise RuntimeError("cannot access 'node_name' before executing action")
         return self.__final_node_name
 
+    def is_node_name_fully_specified(self):
+        keywords = (self.UNSPECIFIED_NODE_NAME, self.UNSPECIFIED_NODE_NAMESPACE)
+        return all(x not in self.node_name for x in keywords)
```

Comment:
@ivanpauno meta, feel free to ignore: if we made this a free function (with all its implications), code would read better e.g.:

```python
if is_node_name_fully_specified(self.node_name):
    # ...
```

Reply:
I don't think it's more readable. I also don't think that the free function will have uses not related with the Node class.

---

This comment applies to https://github.com/ros2/launch_ros/pull/153, in which this PR depends on. Please continue the review there.